### ISSUE TITLE
Feature Request: More Search/Post Processing Abilities

### ISSUE THREAD

**Issue by LeeThompson** [nosmokingbandit/Watcher3#251]
I should probably do these individually but... something none of these apps do that would be very useful would be:

Allow specifying an additional string for a particular movie, say you're after a director's cut or uncut version.   (This could be an extension of the keywords used for qualities but for a specific film.)

Use mediainfo to get actual running time/resolution/frame rate/language information.   This would allow the following features (by film):
- Reject if the wrong cut (i.e. looking for directors' cut of 93 minutes, but this copy is 82m)
- Reject if spoken language(s) are incorrect (with option to accept 'unknown' for those files where it's not identified)
- Reject if subtitle language(s) are incorrect (with option to accept 'unknown' for those files where it's not identified)
- Reject if framerate is a certain set of values or not a certain set of values (i.e. reject if it's 25)

**Comment by nosmokingbandit** [comment]
Originally (like waaaaay back when I only had about 4 people testing this) each movie had its own completely separate quality settings. Going to a shared profile like it is now was what everyone agreed would be best, and trying to keep both was a logistical nightmare.

This may be easier to implement now since I've managed to get the code much more organized than it was back then. Don't get your hopes up, but I'll take a look at it.

**Comment by nosmokingbandit** [comment]
6c5dd0b73cb058d859f7a3cdef450b71aee9ea9d

I added the ability to have specific filter keywords for individual movies. This was not near as complicated as I thought it might be which means I've either done something horribly wrong or I actually kind of know what I'm doing now.

I'm not sure exactly how to identify certain things if it isn't explicitly listed in the release title (which is why scene rules exist). Things like the run time and framerate can be parsed from the file after it has downloaded, but that isn't really a good solution. Scene release rules cover most of this, but that only matters if people actually follow the rules.

**Comment by LeeThompson** [comment]
Yeah framerate etc, no real way to do that except in the post-processing area.   (Although many indexers have a  mediainfo dump of some kind I don't think that's exposed to the API at all.)

**Comment by nosmokingbandit** [comment]
NewzNab *can* include a lot of extra metadata for movies in their api response but I wouldn't trust it. Most indexers only barely follow newznab spec as it is and trying to work around all of the inconsistencies for optional data would be impossible.
